openstudiocore/src/model/Model.cpp

```diff
@@ -1786,6 +1788,21 @@ void addExampleModelObjects(Model& model)
   groundTemp.setNovemberGroundTemperature(19.802);
   groundTemp.setDecemberGroundTemperature(19.633);
 
+  // add SiteGroundTemperatureDeep
+  SiteGroundTemperatureDeep groundTempDeep = model.getUniqueModelObject<SiteGroundTemperatureDeep>();
```

Comment:
Adding these numbers to the example model is fine (although as I said before) it isn't necessary to add every object to the example model.

Adding defaults to the constructor is a little more dangerous.  Unless you are sure that these numbers are better than E+ defaults for almost all models I would stick with using E+ defaults just for traceability.